### PR TITLE
Feat/banner

### PR DESCRIPTION
**Pull request by Pierre-Monier**
- add banner
- refacto close button to be shared accord multiple internal components
- tiny alert refacto for better close button
- add boldText typo for multiple use in internal component

### CODE REVIEW

File: lib/src/components/banners/banner_text.dart [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    final textStyle = dsfrTypography.boldText.copyWith(color: textColor);
+
+    return Text.rich(
+      TextSpan(
+        children: [
+          TextSpan(
+            text: text,
+            style: textStyle,
+          ),
+          if (link != null) ...[
```
Comment (TesteurManiak) [inline lib/src/components/banners/banner_text.dart]: I would recommend to use an internal variable for `link` so you can beneficiate from type shadowing and avoid using the `!` operator after the first null check.

File: lib/src/components/banners/banner.dart [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import 'banner_text.dart';
+
+/// Create a banner .
+///
+/// Specs: https://gouvfr.atlassian.net/wiki/spaces/DB/pages/992903190/Bandeau+d+information+importante
+class DSFRBanner extends StatelessWidget {
+  const DSFRBanner({required this.text, this.onClose, this.link, super.key});
+
+  final String text;
+  final void Function()? onClose;
```
Comment (TesteurManiak) [inline lib/src/components/banners/banner.dart]: You can use the type `VoidCallback?` instead.

**Review by TesteurManiak — COMMENTED** [review]
Everything seems good to me, if you can quickly fix the 2 points I've annoted it'll be perfect. You can also add to the README that the package `url_launcher` is used for the `DSFRBannerText` widget.

**Comment by Pierre-Monier** [comment]
In the readme I wrote that url_launcher is used in `DSFRBanner ` because `DSBRBannerText` is an internal widget, not exposed to the end user

**Pull request merged**